tests/scripts/tmate-pod.yaml

```diff
@@ -0,0 +1,79 @@
+# Apply this manifest to CI clusters to allow debugging complex failures during CI runs
+#
+# Usage:
+#   kubectl apply -f tests/scripts/tmate-pod.yaml
+#   kubectl -n tmate wait --for=condition=ready -l app=tmate pod --timeout=300
+#   sleep 1 # just in case tmate hasn't output its web/ssh links yet
+#   kubectl -n tmate logs deploy/tmate
+
```

Comment:
What happens after the test completes? Does everything get cleaned up? Or how long will the test cluster stay up for debugging?

Reply:
This has no bearing on how long the cluster stays up for debugging

Reply:
Just to be clear, the tmate session would go away as soon as the test completes, right?

Reply:
Or in other words, if the life of the cluster needs to be prolonged, another change to the action would be needed.